capability_linux.go

```diff
@@ -25,21 +26,31 @@ const (
 var (
 	capVers    uint32
 	capLastCap Cap
+	once       sync.Once
 )
 
-func init() {
-	var hdr capHeader
-	capget(&hdr, nil)
-	capVers = hdr.version
-
-	if initLastCap() == nil {
-		CAP_LAST_CAP = capLastCap
-		if capLastCap > 31 {
-			capUpperMask = (uint32(1) << (uint(capLastCap) - 31)) - 1
-		} else {
-			capUpperMask = 0
+func getCapMaskCapLast() (uint32, Cap) {
+	once.Do(func() {
```

Comment:
we actually don't need once.Do here, because initLastCap returns immediately if capLastCap is already set.